### PR TITLE
fix(editor): Replace isInstanceOwner checks with scopes where applicable

### PR DESCRIPTION
**Pull request by cstuncsik**

### CODE REVIEW

File: packages/editor-ui/src/mixins/nodeHelpers.ts [hunk 1]
```diff
@@ -475,11 +476,10 @@ export const nodeHelpers = defineComponent({
 					}
 
 					if (nameMatches.length === 0) {
-						const isInstanceOwner = this.usersStore.isInstanceOwner;
 						const isCredentialUsedInWorkflow =
 							this.workflowsStore.usedCredentials?.[selectedCredentials.id as string];
 
-						if (!isCredentialUsedInWorkflow && !isInstanceOwner) {
+						if (!isCredentialUsedInWorkflow && !this.rbacStore.hasScope('credential:read')) {
```
Comment (alexgrozav) [inline packages/editor-ui/src/mixins/nodeHelpers.ts:482]: Can you use `hasPermission(['rbac'], { rbac: { scope: 'credential:read' } })` to keep consistent?

File: packages/editor-ui/src/components/MainSidebarSourceControl.vue [hunk 2]
```diff
@@ -36,8 +34,11 @@ const tooltipOpenDelay = ref(300);
 const currentBranch = computed(() => {
 	return sourceControlStore.preferences.branchName;
 });
-const isInstanceOwner = computed(() => usersStore.isInstanceOwner);
-const setupButtonTooltipPlacement = computed(() => (props.isCollapsed ? 'right' : 'top'));
+const sourceControlAvailable = computed(
+	() =>
+		sourceControlStore.isEnterpriseSourceControlEnabled &&
+		rbacStore.hasScope('sourceControl:manage'),
```
Comment (alexgrozav) [inline packages/editor-ui/src/components/MainSidebarSourceControl.vue:40]: Can you use `hasPermission(['rbac'], { rbac: { scope: 'sourceControl:manage' } })` to keep consistent?

**Review by alexgrozav — CHANGES_REQUESTED** [review]
Looks good overall. Small changes requested to make sure we keep consistent with permission checks.

File: packages/editor-ui/src/components/SettingsLogStreaming/EventDestinationSettingsModal.ee.vue [hunk 3]
```diff
@@ -306,9 +305,11 @@ export default defineComponent({
 			}
 			return items;
 		},
+		canManageLogStreaming(): boolean {
+			return this.rbacStore.hasScope('logStreaming:manage');
```
[the remark this answers is not in the document]
Comment (cstuncsik) [reply packages/editor-ui/src/components/SettingsLogStreaming/EventDestinationSettingsModal.ee.vue]: Sure, done

File: packages/editor-ui/src/views/SettingsLogStreamingView.vue [hunk 4]
```diff
@@ -158,6 +156,9 @@ export default defineComponent({
 			if (this.disableLicense) return false;
 			return this.settingsStore.isEnterpriseFeatureEnabled(EnterpriseEditionFeature.LogStreaming);
 		},
+		canManageLogStreaming(): boolean {
+			return this.rbacStore.hasScope('logStreaming:manage');
```
[the remark this answers is not in the document]
Comment (cstuncsik) [reply packages/editor-ui/src/views/SettingsLogStreamingView.vue]: Done

File: packages/editor-ui/src/components/MainSidebarSourceControl.vue (hunk 2 above)
Comment (cstuncsik) [reply packages/editor-ui/src/components/MainSidebarSourceControl.vue]: Done

**Review by alexgrozav — APPROVED** [review]

**Review by ivov — APPROVED** [review]

**Review by alexgrozav — APPROVED** [review]

**Comment by janober** [comment]
Got released with [`n8n@1.20.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.20.0)

**Pull request merged**